### PR TITLE
:Wrench: Menu editor language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/prefs/MenuEditorPage.js [hunk 1]
```diff
@@ -963,22 +963,22 @@ var MenuEditorPage = class MenuEditorPage {
...
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Bugs can be reported on <a href="%s">Github</a>.')
+      _('Bugs can be reported on <a href="%s">GitHub</a>.')
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Deep hierarchies are pretty efficient. Put menus into menus in menus!'),
+      _('Deep hierarchies are fairly inefficient. Avoid putting menus into menus in menus.'),
```
Comment (comradekingu) [inline src/prefs/MenuEditorPage.js:981]: This changed. Neither here nor there.

Comment (Schneegans) [reply src/prefs/MenuEditorPage.js:981]: Thank you for this pull request! It definitely improves the language.

However, I do not agree with this particular change. It _is_ a strength of marking menus that you can quickly select items at deeper levels. Consider a menu which has eight root items and then two nested levels of seven items each. Then you have 392 items in total. If you learned the gesture for some of them, you can select them in less than a second. Try this with a conventional linear menu.

**Review by Schneegans — CHANGES_REQUESTED** [review]

File: src/prefs/MenuEditorPage.js (hunk 1 above)
Comment (comradekingu) [inline src/prefs/MenuEditorPage.js:981]:
```suggestion
      _('Keep menu hierarchies efficient by putting the right amount of menus into new ones.'),
```

Comment (comradekingu) [reply src/prefs/MenuEditorPage.js:981]: It is a tradeoff. Tried to do it the right way.

File: src/prefs/MenuEditorPage.js [hunk 2]
```diff
@@ -978,7 +978,7 @@ var MenuEditorPage = class MenuEditorPage {
           .replace('%s', 'https://github.com/Schneegans/Fly-Pie/issues'),
       // Translators: This is one of the hints which are shown in the bottom right corner
       // of the menu editor.
-      _('Deep hierarchies are fairly inefficient. Avoid putting menus into menus in menus.'),
+      _('Keep menu hierarchies efficient by putting the right amount of menus into new ones.'),
```
Comment (Schneegans) [inline src/prefs/MenuEditorPage.js:981]: The point is, that you should prefer depth over breadth when designing menus. For maximum efficiency a menu level should contain at most 8 items. Twelve is OK as well, but eight is better. If you need more, then go deeper!

You could aleays create additional menus, but this is not always desirable as this requires learning more shortcuts.

Maybe something like this?

```suggestion
      _(' Prefer depth over breadth when designing menus. For maximum efficiency, create nested menus when you need more then eight items.'),
```

Comment (comradekingu) [reply src/prefs/MenuEditorPage.js:981]: It is neither obvious that a menu gets broader, nor  what "nesting" actually is.
You either get the point beforehand, or are none the wiser with the explanation.

Comment (Schneegans) [reply src/prefs/MenuEditorPage.js:981]: In my opinion this should be easy to understand. However, I still like the original string "Deep hierarchies are pretty efficient. Put menus into menus in menus!" most. Maybe we should just leave it like this :wink:

**Review by Schneegans — APPROVED** [review]

**Comment by Schneegans** [comment]
I think this is a good solution. Thank you! Do you plan to work on the translation for Fly-Pie 6 today or tomorrow? If not, I would rather reschedule this to the next version and publish the current `release/v6` branch to extensions.gnome.org so that the guys can start the review process.

**Pull request merged**